### ISSUE TITLE
Used text sizes can be too small.

### ISSUE THREAD

**Issue by Golutazem** [misskey-dev/misskey#4896]
In order for text to be easily legible by the user, the text must be at least as large as he has configured. For this reason, absolute sizes should be avoided and any text sizes specified should be greater than 1em or 1rem.
An option for the 'medium' size in the Settings is to not specify any text sizes in CSS and only use what the user has configured for any element.
